tests/mountingOptions/stubs.global.spec.ts

```diff
@@ -300,4 +301,47 @@ describe('mounting options: stubs', () => {
 
     expect(wrapper.html()).toBe('<foo-bar-stub></foo-bar-stub>')
   })
+
+  describe('stub slots', () => {
+    const Component = {
+      name: 'Parent',
+      template: `
+        <div>
+          <component-with-slots>
+            <template #default>Default</template>
+            <template #named>A named</template>
+            <template #noop>A not existing one</template>
+            <template #scoped="{ boolean, string }">{{ boolean }} {{ string }}</template>
+          </component-with-slots>
+        </div>`,
+      components: { ComponentWithSlots }
+    }
+
+    afterEach(() => {
+      config.renderAllStubSlots = false
+    })
+
+    it('renders only the default stub slot by default', () => {
+      const wrapper = mount(Component, {
+        global: {
+          stubs: ['ComponentWithSlots']
+        }
+      })
+      expect(wrapper.html()).toBe(
+        `<div><component-with-slots-stub>Default</component-with-slots-stub></div>`
+      )
+    })
+
+    it('renders all provided stub slots', () => {
+      config.renderAllStubSlots = true
+      const wrapper = mount(Component, {
+        global: {
+          stubs: ['ComponentWithSlots']
+        }
+      })
+      expect(wrapper.html()).toBe(
```

Comment:
what happens if we have

```html
<template>
   <Child>
      Child
  </Child>
</template>
```

or

```html
<template>
   <Child>
      <Another Child>Child</Another Child>
  </Child>
</template>
```

? 

What "should" happen?

An alternative would be to limit this behavior to `shallowMount` - that seems how people usually use it (once we implement shallowMount which should be quite easy with this).

Will we just get `<child-stub />`?

Reply:
I added a test for that case.

Reply:
Cool! Is this in line with the behavior people want/requested in beta (I have no idea what is ideal/desirable; I don't use this feature).

This thread comes to mind; the last user to post has an extremely strong opinion about this: https://github.com/vuejs/vue-test-utils/issues/1307#issuecomment-617491495